app/assets/stylesheets/application.tailwind.css

```diff
@@ -0,0 +1,13 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/*
+
+@layer components {
+  .btn-primary {
+    @apply py-2 px-4 bg-blue-200;
+  }
+}
+
+*/
```

Comment:
- [ ] It's a good convention to not have commented codes in the codebase as they make our codebase dirty and not professional. Kindly remove all commented codes from your codebase so your code looks clean and professional 🙃 😉

Reply:
I forgot about this comment. Thank you!